src/expressions/array-expr.md

```diff
@@ -9,7 +9,7 @@ ArrayExpression -> `[` ArrayElements? `]`
 
 ArrayElements ->
       Expression ( `,` Expression )* `,`?
-    | Expression `;` Expression
+    | Expression `;` GenericArgsConst
```

Comment:
Unfortunately this is not correct, as we accept:

```rust
_ = [0u8; 1 + 2];
```

Thoughts on the right approach?

Reply:
After discussion, we all agreed this should just be `Expression` after all.